formbricks/formbricks
# fix: demo app responsive add favicon

## What does this PR do?

- Make demo app responsive 
- Add favicon

Fixes #2955 


### Screenshots

![Screenshot 2024-08-03 at 11 59 31 PM](https://github.com/user-attachments/assets/17d5fe47-3450-4303-a442-18b8c1b9ace7)
![Screenshot 2024-08-03 at 11 59 01 PM](https://github.com/user-attachments/assets/e26cc49d-665b-48a8-8ec6-810b1cd8eb7b)
![Screenshot 2024-08-03 at 11 58 52 PM](https://github.com/user-attachments/assets/bc58b3f6-dad4-42ed-b96f-e8c4d281713b)


## How should this be tested?
Turn on demo app

1. Check favicon
2. Resize browser size

## Checklist



### Required


### Appreciated
